FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Vladimir Jeremic on 4/27/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedFeedCache
+
+	static func feedImages(from localFeedImges: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
+		return NSOrderedSet(array: localFeedImges.map({ feedImage(from: $0, in: context) }))
+	}
+
+	static func feedImage(from localFeedImage: LocalFeedImage, in context: NSManagedObjectContext) -> ManagedFeedImage {
```

Comment:
This method is not used from anywhere outside of this file thus it can be `private`

Please make it `private`
```suggestion
	private static func feedImage(from localFeedImage: LocalFeedImage, in context: NSManagedObjectContext) -> ManagedFeedImage {
```